use_cases/eluc/app/utils.py

```diff
@@ -308,15 +258,28 @@ def create_pareto(pareto_df: pd.DataFrame, presc_id: int) -> go.Figure:
                                     " Average ELUC: %{y} tC/ha<extra></extra>")
     return fig
 
-
 def load_predictors() -> dict:
     """
-    Loads in predictors from json file according to config.
+    Loads in predictors from disk.
+    TODO: Currently hard-coded to load specific predictors. We need to make this able to handle any amount!
     :return: dict of predictor name -> predictor object.
     """
-    predictor_cfg = json.load(open(os.path.join(constants.PREDICTOR_PATH, "predictors.json")))
-    predictors = dict()
-    # This is ok because python dicts are ordered.
-    for row in predictor_cfg["predictors"]:
-        predictors[row["name"]] = Predictor.SkLearnPredictor(os.path.join(constants.PREDICTOR_PATH, row["filename"]))
-    return predictors
\ No newline at end of file
+    predictors = {}
+    nn_path = "danyoung/eluc-global-nn"
+    nn_local_dir = app_constants.PREDICTOR_PATH / nn_path.replace("/", "--")
+    linreg_path = "danyoung/eluc-global-linreg"
+    linreg_local_dir = app_constants.PREDICTOR_PATH / linreg_path.replace("/", "--")
+    rf_path = "danyoung/eluc-global-rf"
+    rf_local_dir = app_constants.PREDICTOR_PATH / rf_path.replace("/", "--")
+    global_nn = NeuralNetPredictor.from_pretrained(nn_path,
+                                                   local_dir=nn_local_dir)
+    global_linreg = LinearRegressionPredictor.from_pretrained(linreg_path,
+                                                              local_dir=linreg_local_dir)
+    global_rf = RandomForestPredictor.from_pretrained(rf_path,
+                                                      local_dir=rf_local_dir)
+
+    predictors["Global Neural Network"] = global_nn
+    predictors["Global Linear Regression"] = global_linreg
+    predictors["Global Random Forest"] = global_rf
+
+    return predictors
```

Comment:
This function is currently hard-coded to load these predictors. We can handle this more elegantly but for now users aren't submitting models and ultimately we want to be in control of the models that get added, we aren't doing it automatically, so this in theory should be fine.

Reply:
Yes it's fine